provider/provider.go

```diff
@@ -30,6 +30,7 @@ type BaseProvider struct {
 	Watch       bool              `description:"Watch provider"`
 	Filename    string            `description:"Override default configuration template. For advanced users :)"`
 	Constraints types.Constraints `description:"Filter services by constraint, matching with Traefik tags."`
+	Trace       bool              `description:"Display additional provider logs."`
```

Comment:
This only works for 3 providers as of now, right?

How about extending the description like

`Display additional provider logs. (Not supported by all providers.)`